pages/common/az-login.md

```diff
@@ -0,0 +1,29 @@
+# az login
+
+> Log in to Azure.
+> Part of `azure-cli`.
```

Comment:
I'm not sure about this
```suggestion
```

Reply:
I think we added this to the other subcommands which belong to `az`.
We had a [discussion](https://github.com/tldr-pages/tldr/pull/6222/files/a701d72b6205bf20aab0ea714f696ecb995a9778#r669918806) about `azure-cli` & `az` but finally went with `azure-cli`. 
We can revisit if you think this does not make much sense :)

Reply:
What about this?
```suggestion
> Part of `az`, the command line client for Microsoft Azure.
```

Reply:
Thats a little long, but good IMO

Reply:
also edited for our standard